orchestra/gitutils/__init__.py

```diff
@@ -86,8 +86,8 @@ def current_branch_info(repo_path):
     if branch_file.is_file():
         commit = branch_file.read_text().strip()
     else:
-        # Look into .git/info/refs
-        refs = (dot_git / "info" / "refs").read_text().strip()
+        # Look into .git/info/refs and into .git/info/packed-refs
+        refs = (dot_git / "info" / "refs").read_text().strip() + (dot_git / "packed-refs").read_text().strip()
         commit = _only(
```

Comment:
Using `_only` might be problematic in some situations, since it does `assert len(input) == 1`, but there might be the same hash in both `info/refs` and `packed-refs`, I would do:
```python
matching_lines = [...]
assert len(matching_lines) >= 1
assert all(x == matching_lines[0] for x in matching_lines)
commit = matching_lines[0]
```

Reply:
I was under the impression that a ref can be either packed or unpacked, not both.
But what you propose is more robust. On it.

Reply:
See if you like it better now.
I even added some (hopefully) useful messages if the assertions fail.
And I dropped the `_only` helper, that wasn't used anymore at this point.

Reply:
:+1:  on both the comments and removing `_only`